### PR TITLE
Fix "Refresh LUN Size" feature page

### PR DESCRIPTION
**Pull request by rollandf**
- move to "storage" directory
- fix REST code snippet
- Add the UI snapshot as in old wiki
- fix feature status

Signed-off-by: Fred Rolland <frolland@redhat.com>

### CODE REVIEW

**Comment by mureinik** [comment]
:+1:

**Comment by sandrobonazzola** [comment]
:thumbsup:  @thatdocslady can you check if headers/metadata are ok?

**Comment by mureinik** [comment]
@thatdocslady @sandrobonazzola @mykaul this patch is about two weeks old - how can we push it forward?

File: source/develop/release-management/features/storage/lun-resize.html.md [hunk 1]
```diff
@@ -1,28 +1,26 @@
...
-authors: frolland, sandrobonazzola
+authors: frolland, nsoffer
 wiki_category: Feature|LUN_Resize
 wiki_title: Features/LUN Resize
 wiki_revision_count: 23
 wiki_last_updated: 2015-10-14
 feature_name: Refresh LUN Size
 feature_modules: engine/vdsm
-feature_status: On QA
+feature_status: Released
```
Comment (mykaul) [inline source/develop/release-management/features/storage/lun-resize.html.md]: Perhaps we should add in which version?

**Comment by mykaul** [comment]
Small nit on the 'Released' state - I'd add the version in which it was released. Merged without it though.

**Pull request merged**